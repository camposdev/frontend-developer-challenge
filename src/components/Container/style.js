import styled from "styled-components";

export const ContainerComponent = styled.div`
  width: 100%;
  margin: 0 auto;
  padding: 0 15px;

  @media (min-width: 768px) {
    width: 720px;
  }

  @media (min-width: 992px) {
    width: 960px;
  }
`;
