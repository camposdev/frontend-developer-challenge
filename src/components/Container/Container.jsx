import React from "react";
import PropTypes from "prop-types";
import { ContainerComponent } from "./style";

const Container = ({ children }) => (
  <ContainerComponent>
    {children}
  </ContainerComponent>
);

Container.propTypes = {
  children: PropTypes.node.isRequired
};

export default Container;
