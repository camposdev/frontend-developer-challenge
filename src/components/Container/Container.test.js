import React from "react";
import renderer from "react-test-renderer";
import Container from "./Container";

describe("Container test suit", () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Container>Content</Container>)
      .toJSON();
    
    expect(tree).toMatchSnapshot();
  });
});