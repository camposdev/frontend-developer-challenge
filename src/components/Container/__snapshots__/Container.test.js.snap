// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container test suit renders correctly 1`] = `
<div
  className="sc-bdVaJa kHoOvY"
>
  Content
</div>
`;
